sklearn/model_selection/tests/test_validation.py

```diff
@@ -2094,41 +2094,13 @@ def test_fit_and_score_failing():
     failing_clf = FailingClassifier(FailingClassifier.FAILING_PARAMETER)
     # dummy X data
     X = np.arange(1, 10)
-    y = np.ones(9)
     fit_and_score_args = [failing_clf, X, None, dict(), None, None, 0, None, None]
     # passing error score to trigger the warning message
     fit_and_score_kwargs = {"error_score": "raise"}
     # check if exception was raised, with default error_score='raise'
     with pytest.raises(ValueError, match="Failing classifier failed as required"):
         _fit_and_score(*fit_and_score_args, **fit_and_score_kwargs)
 
-    # check that functions upstream pass error_score param to _fit_and_score
-    error_message = re.escape(
-        "error_score must be the string 'raise' or a numeric value. (Hint: if "
-        "using 'raise', please make sure that it has been spelled correctly.)"
-    )
-
-    error_message_cross_validate = (
-        "The 'error_score' parameter of cross_validate must be .*. Got .* instead."
-    )
-
-    with pytest.raises(ValueError, match=error_message_cross_validate):
-        cross_val_score(failing_clf, X, cv=3, error_score="unvalid-string")
-
-    with pytest.raises(ValueError, match=error_message):
-        learning_curve(failing_clf, X, y, cv=3, error_score="unvalid-string")
```

Comment:
This PE doesn't deal with `cross_val_score` and ``learning_curve``. We need to keep this part for now